src/app/pages/categoryPostPage/categoryPostPage.component.html

```diff
@@ -0,0 +1,29 @@
+
+<div name="category" *ngFor="let post of posts">
+<h2>{{post.categoryId.name}}</h2>
+
+<h3>{{post.title}}</h3>
+<br>
```

Comment:
Don't use `<br>`. Instead wrap them in proper divs and use css to define the space between the text.

Reply:
The other Jen is going to address this when she does the css so she can manipulate it the way she wants.